### PR TITLE
Add initial SES module

### PR DESCRIPTION
**Pull request by duboisph**

### CODE REVIEW

**Review by simonrondelez — APPROVED** [review]

File: main.tf [hunk 1]
```diff
@@ -0,0 +1,11 @@
+resource "aws_ses_domain_identity" "domain" {
+  domain = "${var.domain}"
+}
+
+resource "aws_route53_record" "domain_amazonses_verification_record" {
```
Comment (ringods) [inline main.tf:5]: I would leave setting the verification record out of this module. Not all customers will have their DNS hosting on R53.

Comment (duboisph) [reply main.tf:5]: Good point, but then the module is even more "useless". I'll add a flag maybe?

Comment (venturel) [reply main.tf:5]: I'd set it only if we pass the `zone_id`

Comment (duboisph) [reply main.tf:5]: Done.

**Review by ringods — APPROVED** [review]

File: main.tf (hunk 1 above)
Comment (ringods) [reply main.tf:5]: @duboisph I approved, but can you add to the docs that the verification record is added when you pass the zone id?

Comment (duboisph) [reply main.tf:5]: It is...

Comment (duboisph) [reply main.tf:5]: > * \[`zone_id`\]: String(optional): Route 53 zone ID for the SES domain verification. If the `zone_id` is not set, we do not create verification record in R53.

;)

**Pull request merged**